drizzlepac/haputils/cell_utils.py

```diff
@@ -544,10 +546,34 @@ def plot(self, output=None, color='b'):
 
 class SkyCell(object):
 
-    def __init__(self, name=None, projection_cell=None, x=None, y=None):
+    def __init__(self, name=None, projection_cell=None, x=None, y=None, scale="fine"):
         """Define sky cell at position x,y within projection cell.
```

Comment:
This constructor will work as long as either (1) name or (2) (projection_cell + x + y) are specified.  Items (1) and (2) are mutually exclusive.  This looks like it wants an overloaded constructor (which does not exist in Python).  I remember encountering this situation previously, but I cannot find my notes just now.  However, I did find this

https://medium.com/@shamir.stav_83310/overloading-constructors-in-python-17e42b7430b6

which discusses this same topic.  This solution eliminates conflict.

Reply:
This will be addressed in a separate PR to address issue #868 (linked to [HLA-453](https://jira.stsci.edu/browse/HLA-453).